packages/astro-embed-youtube/YouTube.astro

```diff
@@ -33,3 +33,10 @@ const posterURL = poster || `https://i.ytimg.com/vi/${videoid}/hqdefault.jpg`;
 <script>
 	import 'lite-youtube-embed';
 </script>
+
+<style is:global>
+	lite-youtube > iframe {
```

Comment:
We also don’t have to rely on `is:global` here I think. We can scope it to the `<lite-youtube>` in this component and just mark the `iframe` as global:
```suggestion
<style>
	lite-youtube > :global(iframe) {
```

Reply:
thanks for reviewing it @delucis!
The previous suggestion worked perfectly. This however, did not. Just now I've tried applying `:global()` only to `lite-yoube` or `iframe`, as you suggest, but it didn't work. Only if I apply to both, which would be the same as `is:global` I guess.

Reply:
Oh interesting — thanks for testing. Maybe `lite-youtube` removes Astro’s scoping class when the custom element loads or something. `is:global` is probably fine then!